config/optional/core.entity_form_display.node.article.default.yml

```diff
@@ -34,7 +34,6 @@ third_party_settings:
       label: Teaser
       format_type: fieldset
       format_settings:
-        label: Teaser
```

Comment:
I'm not 100% sure, but I think we should keep these labels.

When I check `field_group.field_group_formatter_plugin` configuration schema, it contains `label` property. Schema is defined in `docroot/modules/contrib/field_group/config/schema/field_group.field_group_formatter_plugin.schema.yml`